src/components/map-container.js

```diff
@@ -108,7 +110,35 @@ export default function MapContainerFactory(MapPopover, MapControl) {
       }
     }
 
+    layersSelector = props => props.layers;
+    layerDataSelector = props => props.layerData;
+    mapLayersSelector = props => props.mapLayers;
+    layerOrderSelector = props => props.layerOrder;
+    layersToRenderSelector = createSelector(
```

Comment:
single point of entry to determine which layer should be rendered based on `mapLayer` in splitMaps and `layer.shouldRenderLayer` logic